src/index.js

```diff
@@ -43,9 +43,16 @@ export function rangeFromLineNumber(textEditor: TextEditor, line: number, column
     throw new Error(`Column start (${colStart}) greater than line length (${lineLength})`)
   }
 
+  let colEnd = lineLength
+  const rowText = buffer.lineForRow(lineNumber).substr(colStart)
+  const match = /^( +|[\$\w]+)/.exec(rowText)
```

Comment:
Just so our earlier conversation gets recorded, this line should use something along the lines of:

```js
textEditor.getCursors()[0].wordRegExp()
```